libraries/src/Menu/SiteMenu.php

```diff
@@ -63,7 +63,7 @@ public function __construct($options = array())
 	{
 		// Extract the internal dependencies before calling the parent constructor since it calls $this->load()
 		$this->app      = isset($options['app']) && $options['app'] instanceof CMSApplication ? $options['app'] : Factory::getApplication();
-		$this->db       = isset($options['db']) && $options['db'] instanceof DatabaseDriver ? $options['db'] : Factory::getDbo();
+		$this->db       = isset($options['db']) && $options['db'] instanceof DatabaseDriver ? $options['db'] : Factory::getContainer()->get('DatabaseDriver');
```

Comment:
```suggestion
		$this->db       = isset($options['db']) && $options['db'] instanceof DatabaseDriver
			? $options['db']
			: Factory::getContainer()->get('DatabaseDriver');
```
This should hopefully make PHPCS pass in drone.